ports/esp32s2/mpconfigport.mk

```diff
@@ -31,7 +31,7 @@ CIRCUITPY_ROTARYIO = 1
 CIRCUITPY_NVM = 1
 CIRCUITPY_PS2IO ?= 1
 CIRCUITPY_TOUCHIO_USE_NATIVE ?= 1
-CIRCUITPY_WIFI = 1
+CIRCUITPY_WIFI ?= 1
```

Comment:
What does this do?

Reply:
It means I can place "CIRCUITPY_WIFI = 0" in `makefile` (all lowercase) to override the setting.  Disabling wifi is usually needed to make debug builds fit.